### PR TITLE
Feature/#14 ping command

### LINKED ISSUE #14: Ping command
Implement a command which returns the current ping in milliseconds.

### PR DESCRIPTION
**Pull request by PKlempe**
Resolves #14

### CODE REVIEW

File: bot/__main__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import settings
+
+bot = commands.Bot(command_prefix=settings.BOT_PREFIX)
+initial_extensions = ['bot.util.utilities']
+
+@bot.event
+async def on_ready():
+    """Event handler for Bot entering ready state
+    """
+    print('- {0.user} successfully logged in to Discord!'.format(bot))
```
Comment (SebastiantheAstorian) [inline bot/__main__.py:13]: what does {0.user} do?

File: bot/util/utilities.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    @commands.command(name='ping')
+    async def ping(self, ctx):
+        """
+        Command Handler for the Ping Command
+        Responds with a message containing 'Pong!', as well as the latency to the discord server in ms.
+
+        :param ctx: the context in which the command was called
+        """
+        latency = round(self.bot.latency * 1000, 2)
+        await ctx.send("\U0001F3D3 **Pong!** - {0} ms".format(latency))
```
Comment (SebastiantheAstorian) [inline bot/util/utilities.py:21]: We should probably make a constants file for all unicodes of emojis

File: bot/__main__.py (hunk 1 above)
Comment (PKlempe) [reply bot/__main__.py:13]: https://www.w3schools.com/python/ref_string_format.asp
In general the same as `bot.user`.

File: bot/util/utilities.py (hunk 2 above)
Comment (PKlempe) [reply bot/util/utilities.py:21]: Maybe. I'm kind of unsure how often we're gonna use emojis in our texts.

**Pull request merged**